runtime/lua/vim/lsp/util.lua

```diff
@@ -1367,8 +1367,8 @@ local function close_preview_window(winnr, bufnrs)
     end
 
     local augroup = 'preview_window_' .. winnr
-    api.nvim_del_augroup_by_name(augroup)
-    pcall(vim.api.nvim_win_close, winnr, true)
+    pcall(api.nvim_del_augroup_by_name, augroup)
+    pcall(api.nvim_win_close, winnr, true)
```

Comment:
off-topic for this PR, but this seems like a questionable pattern--it is like `try-catch` without handling the error in any way, which is very troublesome to debug. Maybe we need a util function like `vim.pcall()` that ignores errors containing a specified string, and optionally logs failures.

In other cases, usually there is a condition that can be checked to avoid needing to silence errors.

Reply:
Agreed. 

(A lot of the LSP code seems to date from the very early days of Lua runtime. A general cleanup could be worth it.)

Reply:
`vim.pcall` looks like pretty good. `pcall` can ignore some errors. but sometimes the error information is useful .